### PR TITLE
bug fix for document

### PR DESCRIPTION
**Pull request by wuliupo**
1. add a toggle all button. 2. fix the toggle option if the section is hide. 3. deplicated chart type axis

### CODE REVIEW

File: examples/documentation.html [hunk 1]
```diff
@@ -70,7 +70,6 @@
                 'multiBarHorizontal',
                 'multiBar',
                 'legend',
-                'axis',
```
Comment (wuliupo) [inline examples/documentation.html]: duplicated axis chart

File: examples/stylesheets/styles.css [hunk 2]
```diff
@@ -553,6 +553,7 @@ h3.option {
     padding-left: 10px;
     border-top: 5px solid #CCC;
     margin-top: 10px;
+	min-height: 24px;
```
Comment (wuliupo) [inline examples/stylesheets/styles.css]: bug, if the h3 content is empty, there will be wrong height due to no clear float

File: examples/documentation.html [hunk 3]
```diff
@@ -279,6 +278,9 @@
         <div id="optionjump"></div>
 
         <div style="height: 50px"></div>
+		
+		<h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>
```
Comment (wuliupo) [inline examples/documentation.html]: toggle all chart section

File: examples/documentation.html [hunk 4]
```diff
@@ -17,7 +17,7 @@
     <script>
 
         function expandall(name) {
-            $('#ul_' + name).find("li").click();
+            $('#ul_' + name).show().find("li").click();
```
Comment (wuliupo) [inline examples/documentation.html]: bug, toggle option if the chart section is hide.

**Comment by liquidpele** [comment]
awesome, thanks for contributing!

**Pull request merged**